master_server/dummy_vm_server.py

```diff
@@ -103,7 +112,14 @@ def is_active():
 def flag_status():
 #  Returns the state of VM
   return {'status': flag}
-
+  
+@APP.route('/get_status', methods=['GET', 'POST'])
+def get_status_of_request():
+  task_status_request = Request_pb2.TaskStatusRequest()
+  input_file = request.files['task_request']
+  task_status_request.ParseFromString(input_file.read())
```

Comment:
If we want to serialize to string again in next line why not just return input_file.read()?

Reply:
Yes,I will make that change.

Reply:
Changes has been made as suggested.